etc/replay_data.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import json
+import os
+
+from typing import Any, Iterator
+from datetime import datetime
+
+import boto3
+
+from mypy_boto3_s3.client import S3Client
+from mypy_boto3_sqs.client import SQSClient
+
+
+IS_LOCAL = bool(os.environ.get("IS_LOCAL", False))
+
+
+def into_sqs_message(bucket: str, key: str) -> str:
+    return json.dumps(
+        {
+            "Records": [
+                {
+                    "eventTime": datetime.utcnow().isoformat(),
+                    "principalId": {
+                        "principalId": None,
+                    },
+                    "requestParameters": {
+                        "sourceIpAddress": None,
+                    },
+                    "responseElements": {},
+                    "s3": {
+                        "schemaVersion": None,
+                        "configurationId": None,
+                        "bucket": {
+                            "name": bucket,
+                            "ownerIdentity": {
+                                "principalId": None,
+                            },
+                        },
+                        "object": {
+                            "key": key,
+                            "size": 0,
+                            "urlDecodedKey": None,
+                            "versionId": None,
+                            "eTag": None,
+                            "sequencer": None,
+                        },
+                    },
+                }
+            ]
+        }
+    )
+
+
+def send_s3_event(
+    sqs_client: SQSClient,
+    queue_url: str,
+    output_bucket: str,
+    output_path: str,
+):
+    sqs_client.send_message(
+        QueueUrl=queue_url,
+        MessageBody=into_sqs_message(
+            bucket=output_bucket,
+            key=output_path,
+        ),
+    )
+
+
+def list_objects(client: S3Client, bucket: str) -> Iterator[str]:
+    return (
```

Comment:
this could be more idiomatically (and clearly) written like this:
```python
def list_objects(client: S3Client, bucket: str) -> Iterator[str]:
    for p in client.get_paginator("list_objects_v2").paginate(Bucket=bucket):
        for e in p["Contents"]:
            yield p["Key"]
```
Also, do we need key checks? Those `thing[".."]` will throw `KeyError` if the key is absent.